device-types/Cisco/UBR-10012.yaml

```diff
@@ -0,0 +1,26 @@
+---
+manufacturer: Cisco
+model: uBR10012
+slug: ubr10012
+u_height: 18
+is_full_depth: true
+comments: '[Cisco uBR10012 Data Sheet](https://www.cisco.com/c/en/us/products/collateral/video/ubr10000-series-universal-broadband-routers/product_data_sheet09186a00801ed384.html)'
+module-bays:
+  - name: uplink-slot1
+  - name: uplink-slot2
+  - name: uplink-slot3
+  - name: uplink-slot4
+  - name: rf-slot5
+  - name: rf-slot6
+  - name: rf-slot7
+  - name: rf-slot8
+  - name: rf-slot9
+  - name: rf-slot10
+  - name: rf-slot11
+  - name: rf-slot12
+  - name: power-slot1
+  - name: power-slot2
+  - name: pfe-slot1
+  - name: pfe-slot2
+  - name: timing-slot1
+  - name: timing-slot2
```

Comment:
Position is missing for all of these.  Those should be entered to allow dynamic name creation.